addons/calendar/models/calendar_attendee.py

```diff
@@ -138,7 +138,8 @@ def _send_mail_to_attendees(self, mail_template, force_send=False):
                     partner_ids=attendee.partner_id.ids,
                     email_layout_xmlid='mail.mail_notification_light',
                     attachment_ids=attachment_values,
-                    force_send=force_send)
+                    force_send=force_send,
+                    message_type='auto_comment')
```

Comment:
Nitpicking to ease future changes :) 
```suggestion
                    message_type='auto_comment',
                )
```